js/web/docs/webnn-operators.md

```diff
@@ -0,0 +1,96 @@
+## Operators Support Table
+
+The following table shows ONNX
+operators and the supported opset domain/versions in **WebNN EP** by ONNX Runtime Web. For example,
+`7-12, 13+` means ONNX Runtime Web currently supports opset version 7 to 12, 13 and above.
+
+(**Note**: ONNX Runtime only *guarantees* support for models stamped with opset version 7 or above for opset domain 'ai.onnx'.)
+
+[WebNN API](https://webmachinelearning.github.io/webnn) provides two device types `cpu` and `gpu` to leverage different on-device accelerators. WebNN API implementation in Chromium uses XNNPack backend for `cpu` device type and DirectML backend for `gpu` device type. The op support status behind these two backends is inconsistent since XNNPack backend currently supports limited op scope.
```

Comment:
Regarding to "The op support status", it might be helpful to have a link to https://webmachinelearning.github.io/webnn-status/

Reply:
Good idea👍, link added.